src/main/java/org/stellar/anchor/paymentsservice/Account.java

```diff
@@ -16,7 +16,7 @@
 
     public Network network;
 
-    public AccountType type;
+    public AccountLevel level;
```

Comment:
My rationale was influenced by how Circle deals with accounts: there are distribution accounts and default ones, where the distribution account is the one connected with a bank account (and can make/receive bank payments) while the default ones are regular virtual accounts.

My understanding was that one of them had a higher level than the other, that's why I renamed it globally, but I might be wrong here. Maybe AccountType is more generic and scalable, WDYT? My intention here is to have a way to differentiate the regular accounts from the outliers, like distribution accounts.

Reply:
I think `Type` is more generic, but I don't feel strongly about this.

Will the payments service package eventually support sending and receiving from bank accounts, and if so, will we limit this functionality to a subset of `AccountLevel`'s? I can see this service having many more account levels/types than Circle, since this service will include more payment systems with differing sets of functionality.

Reply:
> Will the payments service package eventually support sending and receiving from bank accounts

Yes.

> and if so, will we limit this functionality to a subset of AccountLevel's?

Firstly, it will be limited by network, accounts from the Stellar network can't send bank payments for instance.
Within Circle, it's limited by account level as well, but the limitation could be different in other networks.

> I can see this service having many more account levels/types than Circle, since this service will include more payment systems with differing sets of functionality.

Giving it another thought... it could be more intuitive to have a `capabilities` object with values like these:
```js
{ (...)
  "capabilities": {
    "send": {
      "circle": true,
      "stellar": true,
      "wire": true,
      "foo": false
    },
    "receive": {
      "circle": true,
      "stellar": true,
      "wire": true,
      "foo": false
    }
  }
(...) }
```

WDYT?

Reply:
Replacing in https://github.com/stellar/java-stellar-anchor-sdk/pull/2/commits/38ca76c3fb8ceb0d136b85558fbd491a7a47c568.

Reply:
Sorry I've been in meetings. This looks great, much more flexible 👍